apps/src/templates/sectionProgressV2/SectionProgressV2.jsx

```diff
@@ -38,13 +38,13 @@ function SectionProgressV2({
 
   return (
     <div className={styles.progressV2Page} data-testid="section-progress-v2">
-      <Heading1>Progress</Heading1>
+      <Heading1>{i18n.progress() + ' (' + i18n.beta() + ')'}</Heading1>
```

Comment:
Please add a new "Progress (beta)" string to the JSON file. Out of all the languages we support there's probably at least one where the word order is different for this phrase, different characters are used instead of parentheses, or something else.